libr/core/panels.c

```diff
@@ -54,9 +51,19 @@ typedef struct {
 	bool refresh;
 } Panel;
 
+enum {
+	PANEL_TYPE_FRAME,
+	PANEL_TYPE_MENU
+};
+
+enum {
+	LAYOUT_DEFAULT,
```

Comment:
Please do not rely on the fact that the first value of enum is usually zero in C. Would be nice to set those those too, not only to compare.

Reply:
btw i've had it fixed. i will send PR again for this